android/app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="app_name">TestApp</string>
```

Comment:
ReactTestApp?

Reply:
ReactNativeTestApp?

Reply:
The iOS app name is ReactTestApp. We should be consistent.

Reply:
iOS should change to ReactNativeTestApp then :)

Reply:
It’s what React Native uses as well. It’s not called ReactNativeInstanceManager 😉

Reply:
But this project is called ReactNativeTestApp, so the name of the app could be the same

Reply:
You could make the same argument for react-native, but look at all the class names in React Native. I think we should make it consistent across the eco system. It’s pretty obvious that this isn’t a React JS app.

Reply:
Did we agree on what we want it to be?